endpoint/endpoint.c

```diff
@@ -46,14 +46,18 @@ endpoint_retcode_t send_transaction_information(int value, const char* message,
   }
 
   size_t msg_len = 0;
+  uint64_t timestamp = time(NULL);
```

Comment:
Instead of invoking `time(NULL)`, [CLOCK_MONOTONIC](http://man7.org/linux/man-pages/man2/clock_getres.2.html) should be taken.